src/app/users/selectOrganisation.js

```diff
@@ -21,16 +21,28 @@ const getNaturalIdentifiers = async (req) => {
   }
 };
 
+const redirectToSelectOrganisationPage = (req, res, model) => {
+  const isManage = req.query.manage_users == true;
+  res.render(
+    `users/views/${isManage? "selectOrganisation": "selectOrganisationRedesigned"}`, 
+    { ...model, currentPage: isManage? "users": "services" }
+  )
+}
+
+
 const get = async (req, res) => {
   await getNaturalIdentifiers(req);
-  return res.render('users/views/selectOrganisation', {
+  
+  const model = {
     csrfToken: req.csrfToken(),
     title: 'Select Organisation',
     organisations: req.userOrganisations,
-    currentPage: 'users',
+    currentPage: '',
```

Comment:
I would probably leave this as 'users' just in case, as a default value

Reply:
done!